README.md

```diff
@@ -210,7 +211,9 @@ namespace TSQLLint.Tests.UnitTests.PluginHandler
             string line;
             var lineNumber = 0;
 
-            while ((line = context.FileContents.ReadLine()) != null)
+            var reader = new IO.StreamReader(File.OpenRead(context.FilePath));

```

Comment:
Instantiating a new `StreamReader` from `File.OpenRead` that has not yet been read allows the `ReadLine` below to work as expected.  Long term it would probably be better to handle this [here](https://github.com/tsqllint/tsqllint/blob/master/source/TSQLLint.Infrastructure/Parser/SqlFileProcessor.cs#L183) so that each plugin doesn't have to do it.